src/painting/painter.rs

```diff
@@ -507,6 +507,16 @@ impl Painter {
         self.stdout.flush()
     }
 
+    /// <placeholder docs>
+    pub(crate) fn clear_display(&mut self) -> Result<()> {
+        self.stdout.queue(cursor::Hide)?;
+        self.stdout.queue(Clear(ClearType::All))?;
```

Comment:
I try all the type and only `All` does what I mean.

Reply:
Have you tried with MoveTo(0,0) first and then try the other ClearType's?

Reply:
Tried. Still almost the same.

`FromCursorDown` and others can clear the buffer for history also. I'd like to get a new screen only, and `All` keeps the buffer history that you can get back by scrolling up.

Reply:
👍🏻